shared/assets/js/departures.js

```diff
@@ -44,6 +81,8 @@ const apiUrl = "https://v6.db.transport.rest/";
             }
             
             document.getElementById("getLocation").disabled = true;
+            document.getElementById("getAddress").disabled = true;
+            document.getElementById("manualAddress").disabled = true;
```

Comment:
You may add a option to clear the results & location so you could search another address afterwards